integration-tests-playwright/playwright/e2e/key-bindings.spec.ts

```diff
@@ -53,3 +53,31 @@ test.describe('diagram - key bindings', () => {
     await expect(page.getByTestId('IconLabel - newBoolean')).toBeAttached();
   });
 });
+
+test.describe('explorer - key bindings', () => {
```

Comment:
I don't know if this test should be here (because it targets the same feature - key bindings), or if I should split the tests of each view as it is proposed in [this PR](https://github.com/eclipse-sirius/sirius-web/pull/5751/files#diff-9dc232adda9b8842b0d48fd1afbf78a3a93b87da339ed02b85ce0c136d515806)